R/oc_bbox.R

```diff
@@ -0,0 +1,146 @@
+#' Bounding box list for opencage queries
+#'
+#' Create a bounding box list for opencage queries.
+#'
+#' @param xmin Minimum longitude (also known as \code{min lon},
```

Comment:
would it be user-friendlier to call the arguments only min_lat etc with no x/y?

Reply:
I chose these for compatibility with ~S~ {sf}, in particular `sf::st_bbox.numeric()`, e.g.
```r
sf::st_bbox(c(xmin = 16.1, xmax = 16.6, ymax = 48.6, ymin = 47.9), crs = sf::st_crs(4326))
```
So a bbox is just a named numeric vector with names `xmin` etc. 

I see your point, but am leaning towards keeping those for compatibility reasons.